components/features/wallet/hooks/use-send-funds.ts

```
@@ -39,40 +41,43 @@ export const useSendFunds = () => {
 			tapasAddress: string;
 			fundedPsbt: string;
 		}) => {
-			setErrorMessage("");
+			setError(null);
 			if (!sighashHexToSign) return false;
 			try {
 				const walletProvider = getProviderStrategy(providerName);
 				const signatureHex = await walletProvider.signTx(sighashHexToSign, {
 					address: tapasAddress,
 				});
+				debugger;
```

Biome (1.9.4)

[error] This is an unexpected use of the debugger statement.

Unsafe fix: Remove debugger statement


(lint/suspicious/noDebugger)